src/runtime/threading_backend.cc

```diff
@@ -124,6 +147,62 @@ int MaxConcurrency() {
   return std::max(max_concurrency, 1);
 }
 
+
+unsigned int configThreadGroup(int mode, int nthreads, std::vector<unsigned int> *sorted_order) {
```

Comment:
no need to return sorted order back, we can just store the order inside the ThreadGroup and use it to support SetAffinity(Configure)